cmd/influxd/server_integration_test.go

```diff
@@ -952,25 +961,23 @@ func runTestsData(t *testing.T, testName string, nodes Cluster, database, retent
 		},
 	}
 
+	// See if we should run a subset of this test
+	testPrefix := os.Getenv("TEST_PREFIX")
```

Comment:
I'm all for it.  I likely won't do it on this PR due to needing to get it shipped.  But if I don't get to it, feel free to add it as well.  Makes a lot of sense to work just like the run flag.